### DISCUSSION TITLE
Mapping high-availability device (firewall HA) in Netbox

### DISCUSSION THREAD

**Discussion by DDmchk** [netbox-community/netbox#11318] [Q&A] [answered]
Hi there!

Can you please suggest optimal way to register high-availabilily pair of devices in active-standby mode in Netbox?
This means literally
1) we have two devices of same HW model with same ports 
2) all ports connected to network infrastructure
3) ports may be assigned to same (VRRP) or different IPs.

I would guess it can be done thorugh virtualization, but do not see my path clearly.

Thank you in advance

**Comment by dreng** [comment, marked as answer]
Hi,

I can't tell if this is the intended way but I did it like this:

- Added device type
- Added 2 devices of just created device type and assigned the unique management IP for each device
- Added a FHRP Group (IPAM => FHRP Groups) with protocol VRRPv3
- Opened the new FHRP Group and added a common Virtual IP Address with role "VRRP"
- Assigned interfaces of both devices to FHRP Group (use +-Button on right hand side of interface entry)
